conf/setup.conf

```diff
@@ -1,23 +1,23 @@
 ##
 ##	You will want to change the following to suit your setup:
 ##
-SUNFLOWERROOT	= /tmp/sunflower-toolchain-github-clone
+SUNFLOWERROOT	= /home/harry/sunflower-simulator
```

Comment:
There is this nifty thing you can do on github reviews where you suggest changes by pressing `ctrl-g`.


```suggestion
SUNFLOWERROOT	= /tmp/sunflower-toolchain-github-clone
```

Reply:
Saying that, because @phillipstanleymarbell opened this pull request, I do not have the power to merge these suggestions via github. Will do in a push to this branch instead.

Reply:
Ah thanks nice to know.